api_v3/services/PlaylistService.php

```diff
@@ -382,7 +382,7 @@ function executeAction( $id , $detailed = false, KalturaContext $playlistContext
 	 * @param KalturaFilterPager $pager
 	 * @return KalturaBaseEntryArray
 	 */
-	function executeFromContentAction($playlistType, $playlistContent, $detailed = false, $pager = null)
+	function executeFromContentAction($playlistType, $playlistContent, $detailed = false, $pager = null, $playlistId = null)
```

Comment:
i dont like the fact that you changed this api action to get playlistId, its whole idea is to create it dynamically